aspnetcore/fundamentals/host/web-host.md

```diff
@@ -217,19 +217,34 @@ WebHost.CreateDefaultBuilder(args)
 
 ### HTTPS Port
 
-Set the HTTPS redirect port. Used in [enforcing HTTPS](xref:security/enforcing-ssl).
+Set the HTTPS port to redirect to if you get a non-HTTPS connection. Used in [enforcing HTTPS](xref:security/enforcing-ssl).
```

Comment:
I could be mistaken, but I don't think this also causes the server to listen on that port.  That is, I think it's possible to accidentally redirect requests to an unused port.  Might be worth noting.